android/app/src/main/java/org/fossasia/openevent/adapters/viewholders/SpeakerViewHolder.java

```diff
@@ -50,9 +55,17 @@ public SpeakerViewHolder(View itemView, Context context) {
         //Attach onClickListener for ViewHolder
         itemView.setOnClickListener(view -> {
             String speakerName = speaker.getName();
+            
+            Bundle bundle = null;
+            if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
+                bundle = ActivityOptions.makeSceneTransitionAnimation((Activity) this.context,speakerImage,speakerImage.getTransitionName()).toBundle();
+            }
             Intent intent = new Intent(this.context, SpeakerDetailsActivity.class);
             intent.putExtra(Speaker.SPEAKER, speakerName);
-            this.context.startActivity(intent);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
```

Comment:
What should happen if SDK version is below `JELLY_BEAN`?

Reply:
@iamareebjamal  Sir , is it recommendable to change the whole way of approaching this as the app might crash at that point when launched in pre Jellybean devices ?

Reply:
Yes, that's what I am concerned about. An if-elseif-else clause would be appropriate. And no need to call me sir :smile: